src/coach/defence/DefencePositionCoach.cpp

```diff
@@ -122,18 +122,18 @@ world::WorldData DefencePositionCoach::removeBotFromWorld(world::WorldData world
     return world;
 }
 Vector2 DefencePositionCoach::getMostDangerousPos(const world::WorldData &world) {
-    if (world.ball->vel.length()>0.5){
-        return world.ball->pos+world.ball->vel*0.3;
+    if (world.ball->getVel().length() > 0.5){
+        return world.ball->getPos() + world.ball->getVel() * 0.3;
```

Comment:
What does 0.5 and 0.3 mean?

Reply:
When reviewing a pull request you should look at the changes made and at the changes that should be made.  My pull request is only about refactoring of the Ball and BallPossession classes, not about the DefencePositionCoach. And because I did not introduce/create these magic values you should not have a remark about them in this pull request.